hrjobcontract/CRM/Hrjobcontract/BAO/HRJobContractRevision.php

```diff
@@ -15,7 +15,9 @@ class CRM_Hrjobcontract_BAO_HRJobContractRevision extends CRM_Hrjobcontract_DAO_
   public static function create($params) {
     global $user;
 
-    $params['editor_uid'] = $user->uid;
+    if (!empty($user->uid)) {
```

Comment:
It wasn't necessary until I've added Unit Test as when running Unit Test we don't have any "signed in" user by default.